cpp/test/sg/hdbscan_test.cu

```diff
@@ -417,9 +422,9 @@ class SoftClusteringTest : public ::testing::TestWithParam<SoftClusteringInputs<
       handle,
       condensed_tree,
       prediction_data_,
-      membership_vec.data(),
       data.data(),
-      raft::distance::DistanceType::L2SqrtExpanded);
+      raft::distance::DistanceType::L2SqrtExpanded,
```

Comment:
Reminder to document the process you followed to get the results on the cpu side so that a future developer can reproduce your results.